### PR TITLE
#660 - used RepoConfigYaml in various tests

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #660 
Extended `RepoConfigYaml` and used it in various test classes to create repo config. 
Decided to leave `DockerProxyTest` as is as (its config is large and such configuration is used only once).

### CODE REVIEW

File: src/test/java/com/artipie/RepoConfigYaml.java [hunk 1]
```diff
@@ -72,6 +75,16 @@ public RepoConfigYaml withPort(final int port) {
         return this;
     }
 
+    /**
+     * Adds port to config.
+     * @param url Port
+     * @return Itself
```
Comment (olegmoz) [inline src/test/java/com/artipie/RepoConfigYaml.java:81]: @olenagerasimova comments look to be off, the method is not about ports

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see juts one minor comment

**Comment by olenagerasimova** [comment]
@olegmoz thanks, corrected

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks, merging

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#670` is not assigned, can't get performer

**Pull request merged**